extension-manifest-v2/src/classes/Account.js

```diff
@@ -84,7 +84,6 @@ class Account {
 			body: data,
 			headers: {
 				'Content-Type': 'application/x-www-form-urlencoded',
-				'Content-Length': Buffer.byteLength(data),
```

Comment:
"Content-Length" will be still automatically filled by "fetch", right?

Reply:
was checking on the backend side and cannot find a proof that this header is in use.

Reply:
I think, it should be still present in the request because fetch adds it implicitly and takes the size of the body.